Server/ServerConfig.cs

```diff
@@ -23,7 +23,7 @@ public class ServerConfig : ConfigLoader<ServerConfig>, IConfigSaver
         /// <summary>
         /// IP of MQTT broker to connect to
         /// </summary>
-        public string BrokerIP { get; set; } = "127.0.0.1";
+        public string BrokerIP { get; set; } = "localhost";

```

Comment:
`127.0.0.1` will be always available. You can modify the `localhost` entry in the host file to point to somewhere else.

Reply:
see the other comment for a reply